cdc/puller/frontier/frontier_test.go

```diff
@@ -198,6 +198,12 @@ func TestSpanFrontierFallback(t *testing.T) {
 	// Bump, here we meet resolved ts fall back, where 10 is less than f.Frontier()
 	// But there is no data loss actually.
 	// f.Forward(spAC, 10)
+
+	spEF := tablepb.Span{StartKey: []byte("e"), EndKey: []byte("f")}
+	f.Forward(0, spEF, 30)
+	require.Equal(t, uint64(20), f.Frontier())
+	require.Equal(t, `[a @ 20] [b @ 20] [c @ 20] [d @ 20] [e @ 30] [f @ Max] `, f.String())
+	require.Equal(t, `[a @ 20] [b @ 20] [c @ 20] [d @ 20] [f @ Max] `, f.SpanString(spCD))
```

Comment:
Why it also contains `[a, b)`, `[b, c)` and `[f, Inf)`? I think it should only contains `[c, d)`.